src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleEntries = sourceString.split(';');
+
+  const stylesObj = styleEntries.reduce((prev, style) => {
```

Comment:
The use of `reduce` here is appropriate for transforming an array into an object. However, ensure that the initial value for the accumulator is a meaningful name that represents the object being constructed, such as `styles` instead of the generic `prev`.